administracion/Working for estability

```diff
@@ -0,0 +1,55 @@
+Working for estability
+
+1. Necesidades Básicas
+	[x] Respirar
+	[x] Beber agua
+	[x] Alimentarse
+	[] Necesidad de dormir
+	[] Necesidad de evitar el dolor
+	[] Necesidad de mantener la temperatura corporal
+		[] Poder quitarme la chamarra cuando tenga calor
+
+2. Necesidades de seguridad y protección
+	[x] Seguridad física
+	[] Seguridad de salud
+		[] Peso
+		[] Aliento
+		[] Condición física
+		[] Fuerza
+		[] Protección de antebrazos
+	[] Seguridad de recursos
+		[] Casa
+		[] Dinero
+		[] Educación
+		[] Automóvil
+		[] Gustos personales
+	[] Necesidad de vivienda
+
+3. Necesidades sociales
+	[] Función de relación
+		[] Amistad
+		[] Pareja
+		[] Colegas
+		[] Familia
+	[] Aceptación social
+
+4. Necesidades de estima
+	[] Estima alta
+		[] Respeto a uno mismo
+		[] Confianza
+		[] Competencia
+		[] Maestría
+		[] Logros
+		[] Independencia
+		[] Libertad
+	[] Estima baja
+		[] Necesidad de atención
+		[] Aprecio
+		[] Reconocimiento
+		[] Reputación
+		[] Estatus
+		[] Dignidad
+		[] Fama
+		[] Gloria
+
+5. Autorealización
```

Comment:
Random file, for another project, that accidentally added to the repository, will be eliminated.